monitoring/port.go

```diff
@@ -103,6 +104,14 @@ func (c *portChecker) checkProcess(proc process, reporter health.Reporter) bool
 		if r.Protocol != proc.socket.proto() {
 			continue
 		}
+		// if prechecks have been run multiple times quickly, sockets from
+		// the ping-pong test may still be lingering in time-wait state and
+		// should eventually go away
+		switch proc.socket.state() {
+		case TimeWait, Close:
```

Comment:
Why close State?

Reply:
I think that it actually means `closed` state so I thought I'd add it for completeness, since it means that the socket is closed and also should be released soon.